compose-method/src/main/java/com/iluwatar/composemethod/App.java

```diff
@@ -0,0 +1,42 @@
+package com.iluwatar.composemethod;
+
+import java.util.Random;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * add original method is rough, and the new add method is using the composed method.
+ * {@link Node} is the node for list, and this term is to add the positive integer in
+ * each node's integer list
+ */
+@Slf4j
+public class App {
+  /**
+   * Program entry point.
+   *
+   * @param args command line args.
+   */
+  public static void main(String[] args) {
+    //initialize a list.
+    Node head = new Node();
+    LOGGER.info("head: {}", head);
+    Random random = new Random();
```

Comment:
You can use `var` keyword here. Please check elsewhere too.

Reply:
Also, use SecureRandom